geometry_msgs/Point[] bottom_lidar_polar
geometry_msgs/Point[] bottom_lidar_cartesian
geometry_msgs/Point[] top_lidar_polar
geometry_msgs/Point[] top_lidar_cartesian
